db/src/main/kotlin/edu/artic/db/daos/ArticExhibitionDao.kt

```diff
@@ -21,4 +21,7 @@ interface ArticExhibitionDao {
 
     @Query("select * from ArticExhibition order by `order` limit 6")
     fun getExhibitionSummary(): Flowable<List<ArticExhibition>>
+
+    @Query("select * from ArticExhibition where id = :id")
+    fun getExhibitionById(id: Int) : Flowable<ArticExhibition>
```

Comment:
`Flowable` will emit if the data changes. do we want this to happen for a lookup on exhibition? If there is nothing, it won't emit. this is a great article here: https://medium.com/google-developers/room-rxjava-acb0cd4f3757

Reply:
we cand o Observable here no problem. as there should be no changes at that point. and I don't even know if I use this rightnow.